schedule/yam/agama.yaml

```diff
@@ -5,3 +5,4 @@ description: >
 schedule:
   - installation/bootloader_start
   - yam/agama/agama_is_visible
+  - yam/agama/patch_agama
```

Comment:
when we patch agama, we only patch the tests...but I see the confusion here because this test module can run even if agama is not visible, the liveiso contains already some test to replace regardless of the state of the web server.
Could you please rename it to `patch_agama_tests.pm` and schedule before `agama_is_visible.pm` ?